### PR TITLE
GET | logout

### PR DESCRIPTION
**Pull request by LinaYahya**
Relates #20 
 - create logout controller
 - test route

### CODE REVIEW

File: server/controllers/auth.js [hunk 1]
```diff
@@ -111,3 +111,7 @@ exports.registerController = async (req, res, next) => {
     }
   }
 };
+
+exports.logout = (req, res) => {
+  res.clearCookie('token').json({ status: 200, message: 'logout success' });
```
Comment (mariamisa) [inline server/controllers/auth.js:116]: Res.status(200).json({statusCode:200,message....})

File: server/controllers/auth.js [hunk 2]
```diff
@@ -111,3 +111,8 @@ exports.registerController = async (req, res, next) => {
     }
   }
 };
+
+exports.logout = (req, res) => {
+  res.clearCookie('token');
+  res.status(200).json({ status: 200, message: 'logout success' });
```
Comment (ranasobeid95) [inline server/controllers/auth.js:117]: by default status code will be `200` so you don't need to use` res.status()`
```suggestion
  res.json({ status: 200, message: 'logout success' });
```

File: test/auth.test.js [hunk 3]
```diff
@@ -71,4 +71,20 @@ describe('login endPoint', () => {
...
+describe('testing for /logout', () => {
+  const token =
+    'token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Mywicm9sZSI6ImFydGlzdCIsImlhdCI6MTU5Mjc0MDcyNn0.MH6UIrt3EhlFnIog8WWXF3tb7LpmELBB8lfD_h15EM4';
+  test('testing for /logout ', (done) => {
+    request(app)
+      .get('/api/v1/logout')
+      .set('Cookie', token)
+      .expect(200)
+      .end((err, res) => {
+        if (err) done(err);
```
Comment (ranasobeid95) [inline test/auth.test.js:83]:
```suggestion
        if (err) return done(err);
```

File: server/controllers/auth.js (hunk 1 above)
Comment (ranasobeid95) [inline server/controllers/auth.js:116]:
```suggestion
  res.clearCookie('token').json({ statusCode: 200, message: 'logout success' });
```

**Pull request merged**